docs/demo/Databricks/Mortgage-ETL-db.ipynb

```diff
@@ -17,7 +17,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 0,
+   "execution_count": null,
```

Comment:
I don't know why this line is modified. What I did is just removing the line containing `hasNan`